### PR TITLE
fix(listing, states): show loading indicator in listing, alignment of subtitle in states

### PR DESCRIPTION
**Pull request by abhishekv24**

### CODE REVIEW

File: packages/apsara-ui/src/Skeleton/Skeleton.styles.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const skeletonClsLoading = keyframes`
 `;
 
 export const StyledSkeleton = styled.div<{ $props?: any; $editor?: boolean }>`
-height:24px;
+// height:24px;
```
Comment (rahmatrhd) [inline packages/apsara-ui/src/Skeleton/Skeleton.styles.tsx:13]: can this be removed if not needed?

File: packages/apsara-ui/src/Modal/Modal.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const Modal = ({
     maskStyle = { background: "rgba(0 0 0 / 0.5);" },
 }: ModalProps) => {
     return (
-        <Dialog.Root open={open} onOpenChange={closable ? onClose : () => {}}>
+        <Dialog.Root open={open} onOpenChange={closable ? onClose : () => ({})}>
```
Comment (rahmatrhd) [inline packages/apsara-ui/src/Modal/Modal.tsx:27]: The previous value in the else condition is `() => void` and the modified one is `() => object`, is this intended? If we don't want to pass anything if `closable` is false, can we use `closable ? onClose : undefined` instead?

File: packages/apsara-ui/src/TableV2/VirtualisedTable.tsx [hunk 3]
```diff
@@ -61,7 +71,7 @@ function VirtualisedTable({ columnsData, sortable = false, rowClick, dataFetchFu
         onSortingChange: setSorting,
         getCoreRowModel: getCoreRowModel(),
         getSortedRowModel: getSortedRowModel(),
-        debugTable: true,
+        debugTable: false,
```
Comment (rahmatrhd) [inline packages/apsara-ui/src/TableV2/VirtualisedTable.tsx:74]:
```suggestion
```

**Review by rahmatrhd — CHANGES_REQUESTED** [review]

File: packages/apsara-ui/src/Skeleton/Skeleton.styles.tsx (hunk 1 above)
Comment (abhishekv24) [reply packages/apsara-ui/src/Skeleton/Skeleton.styles.tsx]: removed

File: packages/apsara-ui/src/TableV2/VirtualisedTable.tsx (hunk 3 above)
Comment (abhishekv24) [reply packages/apsara-ui/src/TableV2/VirtualisedTable.tsx:74]: Let's keep this one in disabled state, as it comes handy to debug table related issues

File: packages/apsara-ui/src/Modal/Modal.tsx (hunk 2 above)
Comment (abhishekv24) [reply packages/apsara-ui/src/Modal/Modal.tsx]: replaced `() => object` with `() => void`

**Review by rahmatrhd — APPROVED** [review]

**Review by StewartJingga — APPROVED** [review]
LGTM

**Pull request merged**